Microsoft.DotNet.Try.Jupyter/Rendering/DefaultRenderer.cs

```diff
@@ -28,19 +28,24 @@ public IRendering Render(object source, IRenderingEngine engine = null)
 
         public IRendering RenderObject(object source, IRenderingEngine engine = null)
         {
-
-            var rows = CreateRows(source, engine);
-            var table = $@"<table>
+            try
+            {
+                var rows = CreateRows(source, engine);
+                var table = $@"<table>
     {rows}
 </table>";
 
-            return new HtmlRendering(table);
+                return new HtmlRendering(table);
+            }
+            catch (Exception)
+            {
+                return new PlainTextRendering(source?.ToString());
```

Comment:
Examples of exceptions we might see here? Should we be displaying them somewhere?